cpmonitor/static/css/main.css

```diff
@@ -0,0 +1,10773 @@
+@charset "UTF-8";
```

Comment:
Is this and `main.css.map` generated by `yarn run compile:css`? If yes, I would rather not check them in, but integrate the generation in the general build process. If we build automatically with GitHub Actions, that can be done, easily.

Reply:
Yes they are generated by the yarn command. I was also a bit torn between committing/not committing them, but decided for committing because then people cloning the repository have one less step to do before starting the app (it already annoyed me a bit that I needed to add the yarn install). But its fine for me if we move that to GitHub Actions.